proto/process/agent.proto

```diff
@@ -1576,6 +1576,48 @@ message DataStreamsAggregations {
 	repeated KafkaAggregation kafkaAggregations = 3;
 }
 
+// A list of the supported operations for the Postgres database.
+// Other operations will not be monitored and will not be included in the agent's output.
+enum PostgresOperation {
+	PostgresUnknownOp = 0;
+	PostgresSelectOp = 1;
+	PostgresInsertOp = 2;
+	PostgresUpdateOp = 3;
+	PostgresDeleteOp = 4;
+	PostgresAlterOp = 5;
+	PostgresCreateOp = 6;
+	PostgresDropOp = 7;
+}
+
+// Represents the statistics for a single Postgres operation.
+message PostgresStats {
+	// The name of the table that this PostgresStats represents.
+	string tableName = 1;
+	// The operation that this PostgresStats represents.
+	PostgresOperation operation = 2;
+	// A protobuf encoded sketch of all the latencies for this operation. In nanoseconds.
+	bytes latencies = 3;
+	// If the PostgresStats has a single sample, this field will be the latency (in nanoseconds) of the only sample.
+	double firstLatencySample = 4;
+	// The number of samples in the latencies sketch.
+	uint32 count = 5;
+}
+
+// Represents the statistics for a single Database operation.
+message DatabaseStats {
```

Comment:
Think about including error codes in the aggregation so that it's ready for future compatibility when you decide to add them